package/MDAnalysis/analysis/accesibility.py

```diff
@@ -0,0 +1,126 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+
+from scipy.spatial import KDTree
+from numpy import arange, pi, sin, cos, arccos
+from .base import AnalysisBase
+
+
+ATOMIC_RADI = {
+    "H": 1.200,
+    "HE": 1.400,
+    "C": 1.700,
+    "N": 1.550,
+    "O": 1.520,
+    "F": 1.470,
+    "NA": 2.270,
+    "MG": 1.730,
+    "P": 1.800,
+    "S": 1.800,
+    "CL": 1.750,
+    "K": 2.750,
+    "CA": 2.310,
+    "NI": 1.630,
+    "CU": 1.400,
+    "ZN": 1.390,
+    "SE": 1.900,
+    "BR": 1.850,
+    "CD": 1.580,
+    "I": 1.980,
+    "HG": 1.550,
+}
+
+
+def atomic_radi(e):
+    """van der Waals radii"""
+    DEFAULT_ATOMIC_RADI = 2.0
+    return ATOMIC_RADI[e] if e in ATOMIC_RADI else DEFAULT_ATOMIC_RADI
+
+
```

Comment:
Can you please add the DOI(s) of the relevant publication(s)/source(s) on which this analysis class is based? People need such references to double check the method and see where the default values come from.

You can find an example of how to do that here: https://github.com/MDAnalysis/mdanalysis/blob/d27a32abe39cc950a3943f3c1a03340f86c22a4a/package/MDAnalysis/analysis/msd.py#L256-L263